build/Xml.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.IO;
+using System.Xml;
+using Nuke.Common;
+using Nuke.Common.IO;
+
+public static class Xml
+{
+    public static string XmlPeek(AbsolutePath filePath, string xpath)
```

Comment:
Cake has a native XmlPeek, but Nuke does not.
In the future, we could either submit a PR to Nuke, or have our own `Octopus.NukeHelpers.nupkg` with this in it (and that gets even nicer when we tie that in with dependabot - all our builds get these updates automatically rather than the copy-paste ickyness we have today with cake)